docs/pages/usage-billing.mdx

```diff
@@ -50,43 +50,48 @@ calculate three types of billing metrics:
 
 ### Monthly Active Users 
 
-Monthly Active Users (MAU) are calculated as the aggregate number of unique
-active users (as defined in the Services Agreement and aggregated over each
-monthly period starting on the Subscription Start Date and ending on each
-monthly anniversary thereafter) accessing Teleport. 
-
-"Active" means a user having performed any auditable activity in a resource
-(e.g., an action that would appear in an audit log such as logging into a
-resource via the Web UI or via `tsh login`, submitting an Access Request, etc.).
-The MAU calculation does not include automated actions, such as the modification
-of a user's role by an administrator or the automatic creation of a user
-automatically through an identity provider.
+The Monthly Active Users (MAU) is the aggregate number of unique
+active users accessing Teleport. 
+
+We aggregate MAU over each monthly period starting on the subscription start date and ending on each
+monthly anniversary thereafter.
+
+"Active" means a user having performed any activity that would appear in an audit; for example, 
+connecting to a resource via the Web UI or via `tsh login`, submitting an Access Request and so on.
```

Comment:
```suggestion
"Active" means a user having performed any activity that would appear in a Teleport audit log, for example, 
connecting to a resource via the Web UI or via `tsh login`, submitting an Access Request, and so on.
```
- Minor grammar/style nits
- Adding "log" to clarify that we're referring to the Teleport audit log. I'm assuming this is correct—if not, we'll want to clarify what "an audit" means